src/tree/tree.scss

```diff
@@ -17,6 +17,28 @@ $iui-expander-button-width: ($iui-icons-default) + ($iui-expander-inline-padding
   list-style: none;
 }
 
+@mixin iui-tree-item {
+  outline: none;
+
+  &:focus > .iui-tree-node {
+    @include themed {
+      outline: 1px solid t(iui-color-foreground-primary);
+    }
+    outline-offset: -1px;
+
+    &.iui-active {
+      @include themed {
+        outline: $iui-xxs solid t(iui-color-foreground-primary);
+      }
+      outline-offset: -2px;
+    }
+  }
+
+  &:focus:not(:focus-visible) > :where(.iui-tree-node:not(.iui-active)) {
```

Comment:
I used `:where` to reduce specificity. This should be removed with IE support drop.